src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/entities/DraftAdjudication.kt

```diff
@@ -11,8 +11,9 @@ import javax.persistence.Table
 @Table(name = "draft_adjudications")
 data class DraftAdjudication(
   override val id: Long? = null,
-  val reportNumber: Long?,
   val prisonerNumber: String,
+  val reportNumber: Long? = null,
+  val reportByUserId: String? = null,
```

Comment:
How were we getting this before? Were we keyed of the IncidentDetails object?

Reply:
No, no, we used the `createdByUserId` from the DB audit tables - that is why we have this bug